OutOfSchool/OutOfSchool.Common/Validators/CustomUkrainianNameAttribute.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+using OutOfSchool.Common.Extensions;
+
+namespace OutOfSchool.Common.Validators;
+
+/// <summary>
+/// Validation attribute for Ukrainian name.
+/// </summary>
+public class CustomUkrainianNameAttribute : DataTypeAttribute
+{
+    public CustomUkrainianNameAttribute()
+        : base(DataType.Text)
+    {
+    }
+
+    /// <summary>
+    /// Checks that the value is valid Ukrainian name.
+    /// </summary>
+    /// <remarks>
+    /// This method returns <c>true</c> if the <paramref name="value" /> is null.
```

Comment:
Please consider rephrasing text 
`This method returns <c>true</c> if the <paramref name="value" /> is null.` ...or empty